repos/gems/include/nano3d/sqrt.h

```diff
@@ -12,7 +12,7 @@
  */
 
 #ifndef _INCLUDE__NANO3D__SQRT_H_
-#define _INCLUDE__NANO3D__SQRT_H__
+#define _INCLUDE__NANO3D__SQRT_H_
 
```

Comment:
Oh, that's embarrassing.

Reply:
Its probably harmless, but its nice clang warns about problems like this.